Riesen wanted tasers un-nerfed.

Skrylar opened the pull request:
Changed range of electrodes and doubled stun times.

gcoremans committed:
Fixes adminjump being dumb.

InfiniKraut commented:
Uh, I'm not really managing git anymore, you'll have to do a pull request at whatever git repo the new server uses

Skrylar commented:
I'll see if I can figure out whoever that is now.

Commit:
Allows paper burning

Commit:
Restores gibbing for suiciding with C4

phil235 committed:
Fixes HydroTray Lights sprites.

Cheridan committed:
Merge pull request #7449 from RemieRichards/NecromanticStone

Adds a Necromancy Stone

Cheridan committed:
Merge pull request #8123 from Ikarrus/randomspawns

Randomizes Order of Job Spawn Points

Cheridan committed:
Merge pull request #8125 from Ikarrus/assistantspawns

[Map] Shuffles Assistant Spawn Locations

Cheridan committed:
Merge pull request #7986 from phil235/TaserJitterFix

Fixes taser stun duration

Cheridan committed:
Merge pull request #8106 from Ikarrus/preserveobjectives

Preserve Objectives when admins remove antag status

Cheridan committed:
Merge branch 'BlobReroll' of git://github.com/RemieRichards/-tg-station into RemieRichards-BlobReroll

Conflicts:
	code/game/gamemodes/blob/overmind.dm

Cheridan committed:
Merge branch 'RemieRichards-BlobReroll'

Cheridan committed:
Merge pull request #8147 from Cheridan/master

Manual Merge of #7607

Incoming5643 committed:
When a round is converted, late join antagonists for that roundtype will be made for the rest of the round.

Skrylar committed:
Merge remote-tracking branch 'kraut/master'

These are the last updates Nel curated.

Commit:
Fixes #8116 trails bug

Cheridan committed:
Merge pull request #8101 from MrPerson/alert_alert_ur_a_faget

On screen alert system

Jordie0608 committed:
Improves drill's handling of power

Razharas committed:
Merge pull request #8131 from Jordie0608/sooneverythingwillbechangelogs

Re-adds changelog popup as a config option

Jordie0608 committed:
Taser changelog and spellfix

Jordie0608 committed:
Changelog compiled

Jordie0608 committed:
Edited example yml for only double quotes

Jordie0608 committed:
rogue or

gcoremans committed:
Moves chat toggles into their own variable.

gcoremans committed:
Merge pull request #8156 from Mandurrrh/trailsbugfix

Fixes #8116 trails bug

gcoremans committed:
Merge pull request #8132 from MrStonedOne/wellthen

Fixes irc alerts of adminhelps not showing the ckey

WJohn committed:
Merge pull request #8075 from Xhuis/the_coderbus

Adds the Coderbus and Adminbus bar signs: No merge conflicts edition

gcoremans committed:
Merge pull request #8013 from paprka/altclick

Alt-click for aesthetic toggling

gcoremans committed:
Merge pull request #8014 from paprka/altclick2

Alt click for PDAs

gcoremans committed:
Merge pull request #8065 from paprka/squiddeath

Rebalances thermals

Commit:
adds more barsigns

Incoming5643 committed:
if you have AI or cyborg set to high, you'll see that in the preview window instead of a nondescript naked human

AnturK committed:
Fixes pref flags

Cheridan committed:
Merge pull request #8146 from phil235/HydroTrayLightsSpritesFix

Fixes HydroTray Lights sprites.

Cheridan committed:
Merge pull request #8128 from Ikarrus/gangfixup

Makes Gang mode functional again

MrsTonedOne committed:
IsBanned() now rejects blank ckeys/ips/cids

Also removes the admin notice. Its still logged, but this way the admins don't get spammed during world reboot.

Cheridan committed:
Merge pull request #8140 from phil235/doughbeakerfix

Improvised heating fixes

Cheridan committed:
Merge pull request #8145 from Incoming5643/dangit_bobby_its_all_ones_and_zeros

Hotfix for Blob mulligan rounds not ending | Nuke codes being given in times of peace | AI desiring to blow up the station

Cheridan committed:
Merge pull request #8114 from phil235/CookTrash

Cooking fixes for meat and trash

Cheridan committed:
Merge pull request #8076 from xxalpha/lighters

Grayscale Lighters

Cheridan committed:
Merge pull request #8137 from Incoming5643/so_misunderstood

Removes the Spooky

phil235 committed:
Fixes ghostburger and appendixburger icons.

PlayerDeer committed:
Merge pull request #8098 from pudl/fire

resprites fire extinguishers

Cheridan committed:
Merge pull request #8108 from TheVekter/vodkasprite

Fixes the broken Badminka Vodka sprite

Cheridan committed:
Merge pull request #8163 from Jordie0608/hiorangeshowareyou

Changelog compiled and instructions edit

Commit:
changelog

phil235 committed:
Fixes maintenance drone being targeted by the AI's porta turrets.
Fixes occupied mechs not being targeted by AI's porta turrets.
Simplifies target searching a bit.
Non AI Porta turrets with check_anomalies now also targets living/carbon/slime.

Cheridan committed:
Merge pull request #8080 from Incoming5643/Appearance_fix

Tweaks and fixes for the appearance window

phil235 committed:
Fixes vines not releasing mobs from their grip when destroyed.

Cheridan committed:
Merge pull request #8126 from Iamgoofball/raginmages

fixes Ragin' Mages making only 1 mage

Cheridan committed:
Merge pull request #8169 from dannno/morebarsigns

adds more barsigns

Commit:
Merge pull request #8107 from Cheridan/loot_ops

Nuke Op Loot Pinata Tweaks

Commit:
Merge pull request #7751 from RemieRichards/ProceduralGeneration

Procedural Map Generator System

Cheridan committed:
Merge pull request #8142 from phil235/girderfixes

Fixes many checks in construction from stacks,

Incoming5643 committed:
byond inline spell checker fucking when

Cheridan committed:
Merge pull request #8121 from Incoming5643/everything_I_code_never_works

Fixes pop cap messages not properly showing custom messages (with default config.txt)

Cheridan committed:
Merge pull request #8122 from Xhuis/miners_messiah

Adds the ore satchel of holding

Menshin committed:
* Fixes runtimes with Summon Guns.
* Fixes some summoned guns uncorrectly instanciating pins

Commit:
Fixes lightup floortiles icon state changing with merge.

Iamgoofball committed:
Nerfs acid blob again

goddamn acid blob

RemieRichards committed:
Merge pull request #8135 from Miauw62/adminjumpfixes

Fixes adminjump being dumb.

phil235 committed:
Fixes disposal climbing.
Aliens, slimes and monkeys can climb into disposal but cannot flush it.
Non carbon mobs can no longer climb into disposal by themselves.
Stuffing another mob in a disposal unit is now only done via grabbing.(fixes bugs like stuffing someone while being inside the disposal yourself)
Large mobs no longer fit inside disposal.
Fixes hydrotray growing overlay layer.
Fixes slicing broken pipes dropping a straight pipe.
Fixes typos.
Removing unused var from disposal.

RemieRichards committed:
Hostile mobs no longer run all their hostile targeting/hunting code if there is nobody (or item, for those that hunt items) near them, they simply move to a simpler loop that decides whether to let the main loop continue.

phil235 committed:
Ventcrawlers can now climb into disposal by themselves.

phil235 committed:
Make Airlocks with safety on no longer close on dense objects.

phil235 committed:
Monkeys and xeno can flush disposal units.

Iamgoofball committed:
Black powder hotfix

seriously merge this ASAP, fuck your 24 hour limit this needs fixed

RemieRichards committed:
Merge pull request #8208 from Iamgoofball/patch-42

Black powder hotfix

RemieRichards committed:
Merge pull request #8102 from Mandurrrh/tableclimbnerf

Balancing out tableclimbing.

Commit:
removes changelog because I goofed its a bug fix

phil235 committed:
Changing Coffee jittering to be an overdose effect.

Iamgoofball committed:
Update Blob-Reagents.dm

Commit:
Fixes ogresight + removes burntime var

Commit:
Update paper.dm

Commit:
fixes args

Commit:
testing

Commit:
A bit of a test I did, added slimecake.

Commit:
Updated

Commit:
Updated

Commit:
Unticked

Jordie0608 committed:
Fixes and adds missing ymls

Jordie0608 committed:
Changelog compiled

Jordie0608 committed:
Moves and edits readme

Commit:
Added a changelog (you're welcome Jordie)

Cheridan committed:
Merge pull request #8188 from phil235/VineGripFix

Fixes vines not releasing mobs from their grip when destroyed.

Incoming5643 committed:
Adds a magic mirror to the wizards shuttle. The magic mirror has greatly expanded functionality compared to the normal one, and can additionally change a wizards name, gender, race, hair color and eye color.

Additionally this fixes up some rogue simulated airless tiles on the wizards shuttle (these tiles were why the creature would always end up dead)

gcoremans committed:
Merge pull request #8216 from Jordie0608/newmemeisgrudgechangeloggingremierichards

Changelog

phil235 committed:
Fixes hulks not losing their mutation when low on health.
Fixes carbon mobs resistant to paralysis not going unconscious when in crit.

norill committed:
fixed improper req_tech "power" -> "powerstorage" in several research designs

Iamgoofball committed:
Cyborgs no longer have flesh to knit.

Commit:
Fixed changelog and color

Iamgoofball committed:
Update Chemistry-Goon-Medicine.dm

Incoming5643 committed:
Fixes cavity implant surgery not respecting NODROP

phil235 committed:
Changes how doughs are made. 10water+15flour chem reaction for dough. 15flour+15eggyolk+5sugar for cake batter (or alternatively for vegans 15 soymilk instead of the eggyolk).
Introduce egg yolk reagent
You can break eggs in open reagent containers or blend them to get egg yolk.
Fixes breadslices sprites and other merge conflict bugs in food.dmi.
Change customizable snack max volume to 60. Buff nutriment amount in doughs.

Commit:
Loyalty Implant Firing Pins

- Allows the construction of Loyalty Pins at Combat tech 5, instead of
10.

Commit:
Changelog

Mandatory sacrifice to Jordie!

Cheridan committed:
Merge pull request #8197 from Menshin/summon_guns_fix

Some fixes with Summon Guns

Cheridan committed:
Merge pull request #8205 from phil235/AlienDisposalFix

Fixes disposal climbing.

Cheridan committed:
Merge pull request #8186 from phil235/ghostburgerfix

Fixes ghostburger and appendixburger icons.

Incoming5643 committed:
Explicitly passes a mobs mutant_color in blood samples

Cheridan committed:
Merge pull request #8185 from phil235/TurretMechFix

Porta turret fixes

Cheridan committed:
Merge pull request #8210 from phil235/CoffeeOverdose

Changing Coffee jittering to be an overdose effect.

Cheridan committed:
Merge pull request #8207 from phil235/AirlockDensityObstacleFix

Make Airlocks with safety on no longer close on dense objects.

Cheridan committed:
Merge branch 'renamingandreskinning' of git://github.com/paprka/-tg-station into paprka-renamingandreskinning

Conflicts:
	icons/mob/back.dmi

Cheridan committed:
Merge branch 'paprka-renamingandreskinning'

Cheridan committed:
Merge branch 'master' of https://github.com/tgstation/-tg-station

Cheridan committed:
Merge pull request #8248 from Cheridan/master

Manual Merge of #7983

Cheridan committed:
Merge pull request #8162 from Jordie0608/thisdrillwillnotpiercetheheavensbecauseithasbroken

Fixes #8148, Improves drill's handling of power

Cheridan committed:
Merge pull request #8141 from Incoming5643/Subjectivity_is_a_bitch

Skipping mulligan antag based on Time/Subjectivity from admins + tweaks

Cheridan committed:
Merge pull request #8177 from MrStonedOne/patch-2

IsBanned() now rejects blank ckeys/ips/cids

Cheridan committed:
Merge pull request #7984 from Incoming5643/farewell_to_shitty_slimes

Reworks the pink/light pink slime line to not be nearly worthless to breed

Cheridan committed:
Merge pull request #8073 from Xhuis/burn_this_message_after_you_read_it

Adds paper burning

Cheridan committed:
Merge branch 'master' of https://github.com/tgstation/-tg-station

Cheridan committed:
Merge branch 'TheAnimalsTheyAttack!' of git://github.com/Sometinyprick/-tg-station into Sometinyprick-TheAnimalsTheyAttack!

Conflicts:
	code/game/gamemodes/wizard/spellbook.dm
	icons/mob/mask.dmi

Cheridan committed:
Merge branch 'Sometinyprick-TheAnimalsTheyAttack!'

Cheridan committed:
Merge pull request #8251 from Cheridan/master

Manual Merge of #7776

Cheridan committed:
Merge pull request #8016 from RemieRichards/EmbeddingBalance

Embedded Item Balancing. Also fixes #8029

Cheridan committed:
Merge pull request #8202 from Mandurrrh/fixinlightfloors2

Fixes lightup floortiles icon state changing with merge.

Cheridan committed:
Merge pull request #8100 from Incoming5643/newbieguard

Adds a free pass to assistant for player accounts younger than two weeks

Cheridan committed:
Merge pull request #8215 from Ahammer18/Ahammer18

Slime Cake

Jordie0608 committed:
unhardcodes brig timers

Steelpoint committed:
InitalCommit

phil235 committed:
Simplifies laser reflection code, no more reflect_chance var at the obj/item level.

Commit:
Fixes "Sensor Augmentation" showing in silicon verbs

- Fixes #8167

RemieRichards committed:
Merge pull request #7872 from AnturK/hiss

Makes NPC queens/drones plant weeds/eggs

phil235 committed:
Fixes Strangeness mutation gain text not appearing.

Incoming5643 committed:
Adds a cheap looking pulse animation for magic mirrors so wizards notice they are actually more than normal mirrors

Adds a changelog

phil235 committed:
Fixes a runtime with buckling screen alert
Fixes being able to buckle to janicart while crit and when there's a dense object on the janicart's turf.
Fixes alien nest not buckling and being wrongly pixel shifted, and fixed a runtime.
Fixes golems being super easy to weaken by punch and not doing more damage themselves via punch.
Fixes some typos.

phil235 committed:
slight fix

phil235 committed:
Changes egg breaking to use reagent_containers/glass/attackby() (fix issue with custom egg salad)
You can only break eggs on glass containers (beaker,bucket,bowl,etc)

phil235 committed:
Merge branch 'master' of https://github.com/tgstation/-tg-station into DoughChange

Conflicts:
	icons/obj/food.dmi

phil235 committed:
Fixes book bags not being able to hold bibles.
Changes the book bag sprite so it looks more like a briefcase and isn't confused with a backpack.

RemieRichards committed:
A restructure/removal/cleanup of Ninja Code

Commit:
Removes unnecessary stuff

RemieRichards committed:
Adds SLOT_BACK for energy katana, makes energy katana call emag_act() on what it strikes (mirroring energy/blade functionality cleanly), Removes energy/blade hacks.

MMMiracles committed:
ASS BLAST USA USA USA USA

MMMiracles committed:
changelog

RemieRichards committed:
Moves ninja shoes into the ninja folder

MMMiracles committed:
adds the freedom eagle, the only true friend a patriot can find within the region of socialist space.

MMMiracles committed:
Merge branch 'usaneverdie' of https://github.com/MMMiracles/-tg-station into usaneverdie

Cheridan committed:
Merge pull request #8138 from RemieRichards/BadminFuckingWithIcons

Clothing Icon File Admin Editing

Incoming5643 committed:
The roundtype will be converted. If you feel that the round should not continue, end the round now.
The roundtype will be converted. If you feel that the round should not continue, end the round now.
The roundtype will be converted. If you feel that the round should not continue, end the round now.

Cheridan committed:
Merge pull request #8269 from Incoming5643/The_roundtype_will_be_converted

[#savethebadmins] Fixes the admin message spam in malf

Incoming5643 committed:
Hamfistedly fixes #8266

Steelpoint committed:
AdditionalSpelling

phil235 committed:
Fixes resisting out of cuffs failing.

Jordie0608 committed:
step 1

Jordie0608 committed:
compiled

Jordie0608 committed:
Search button on game panel spawn menu

Jordie0608 committed:
remie's correction

MMMiracles committed:
typo fix

Commit:
Removes old unused file I forgot about

MrPerson committed:
Pulling icon en-speedening

Made the little pulling icon on the bottom right corner of your screen update right away when you start or stop pulling instead of updating as part of life(). Or Move().

gcoremans committed:
Reduces the taser firing delay.

phil235 committed:
Fixes steel cap bug. Steel cap now growns steel logs similar to wooden logs, you use the hatchet to get rods.
Reorganizes all flashing to use the same mob proc 'flash_eyes()' (welding tool use, flashbang, flashers, laserpointers, etc).
Fixes making sandstone not stacking the result sandstones on the floor.
Sandstone, grass tile and carpet tile now look for other incomplete stack on their creation location to try to put them all in one stack (exactly like wood planks).
Fix typos.
Remove banglet var from all flashbangs, making flashbangs from clusters behave exactly like the other ones.
Renames eyecheck()proc to check_eye_prot() and moves it to mob/living level, elso creates check_ear_prot() for checking ear protection.

phil235 committed:
slight adjustments..

phil235 committed:
Merge pull request #8254 from Steelpoint/HoSGunSpelling

[BALANCE]Buffs the Head of Security's Firearm.

phil235 committed:
Merge pull request #8134 from Miauw62/ghostradios

Adds a toggle so ghosts can hear radio chatter more easily.

Commit:
Changed "Toggle Suit Sensors" to "Adjust Suit Sensors" because we now have a menu that pops up when you want to change your suit sensors.

phil235 committed:
Merge pull request #8262 from MMMiracles/usaneverdie

Liberation Station 3: Patriotic Tears

Commit:
Added drone asses

Commit:
Added drone asses

Skrylar committed:
Merge remote-tracking branch 'upstream-tgstation/master' into sup-tg

Conflicts:
	code/game/objects/items/weapons/tanks/watertank.dm
	code/modules/mob/living/simple_animal/friendly/drone.dm
	code/modules/projectiles/projectile/energy.dm
	config/admins.txt
	config/config.txt
	config/game_options.txt

Skrylar committed:
Patch drone uprisings in to /tg/'s drone code.

Jordie0608 committed:
Merge branch 'master' of https://github.com/tgstation/-tg-station into imrunningoutofwittymessagesforthese

Jordie0608 committed:
updated compile

Cheridan committed:
Merge pull request #8272 from Jordie0608/imrunningoutofwittymessagesforthese

Changelog

Cheridan committed:
Merge pull request #8275 from JJRcop/whoops

Removes old unused file I forgot about

norill committed:
acquiring negative mutations shows in red in chat

norill committed:
Revert "acquiring negative mutations shows in red in chat"

This reverts commit 5e003e828d97e8463edbca7ca1085b4ebf9ed314.

norill committed:
acquiring negative mutations shows in red in chat

phil235 committed:
Fixes notasandwich and flamethrower tablecraft recipes
Add a recipe detection to tablecrafting, if you have some of the things required for the recipe (50% of the amount of different things needed, i.e if less than 3 different things in the recipe then you need 1, otherwise 2+), it will now appear in grey in the tablecrafting window. The tablecrafting window will also lists the requirements for each recipes it shows.

hornygranny committed:
Merge pull request #8287 from phil235/CuffResistFix

Fixes resisting out of cuffs failing.

xxalpha committed:
Fixes not being able to move from a single lattice.

phil235 committed:
Door can now crush mech, dealing damage, just like with mobs.

phil235 committed:
Fixes burger and sandwich ingredient stacking and sprites.

phil235 committed:
Fixes russian revolver's special mechanism message appearing when placing it on a table or in your inventory.
Fixes its description about auto spinning barrel.
Fixes its behavior when used as a melee weapon.

phil235 committed:
Fixes chemsprayer duplicating reagents. Simplifying its code.
Chem puff spray can no longer give more chems to mobs than it contains.
Mobs being in front of the chem puff shorten its total range.
Fixes using the spray at short range not showing the whole chemical puff animation.
For a given max range of a spray, aiming closer will now transfer more chemicals to each mobs/objs/turfs than aiming the furthest.

phil235 committed:
Fix for flamethrower deconstruction.

phil235 committed:
Merge pull request #8294 from norill/mutfix

acquiring negative mutations shows in red in chat

MMMiracles committed:
Adds the ability to spike bears on the meatspike for their pelt, as well as yield 5 meat instead of 3. Less messy, more efficient.

MMMiracles committed:
bop

MMMiracles committed:
changelog

Cheridan committed:
Merge branch 'NinjaCleanup' of git://github.com/RemieRichards/-tg-station into RemieRichards-NinjaCleanup

Conflicts:
	icons/mob/back.dmi

remie is a nerd btw

Cheridan committed:
Merge branch 'RemieRichards-NinjaCleanup'

Cheridan committed:
Merge pull request #8322 from Cheridan/master

Manual Merge of #8261

Cheridan committed:
Merge pull request #8270 from Incoming5643/you_cannot_reblob

Keeps badmin blobs from trying to win with blob round blobs objectives.

Cheridan committed:
Merge pull request #8253 from Jordie0608/mergeconflictsihavenoideawhatyouretalkingabout

Unhardcodes brig timers

MMMiracles committed:
whoops

phil235 committed:
Merge pull request #8232 from Iamgoofball/patch-43

Cyborgs no longer have flesh to knit.

phil235 committed:
Merge pull request #8230 from norill/master

fixed improper req_tech "power" -> "powerstorage"

phil235 committed:
Merge pull request #8235 from Incoming5643/you_wouldnt_download_a_car

Fixes cavity implant surgery not respecting NODROP

gcoremans committed:
Chat toggles are saved to the savefile properly again. Fixes #8318

Commit:
Increases requirements

Reqs to make it are now Combat 6, Mats 6, Power 3.

MrPerson committed:
Code cleanup
Removed a bunch of unused or hardly used vars on /mob and /atom/movable.
Added defines for mob/living/var/mob_size so it's not using magic numbers.

Cheridan committed:
Merge pull request #8259 from phil235/BibleBookBagBug

Fixes book bags not being able to hold bibles and looking like backpacks

Cheridan committed:
Merge pull request #8237 from GunHog/CherryFlavoredPins

Loyalty Implant Firing Pins

Cheridan committed:
Merge pull request #8283 from Miauw62/taserdelayymao

Reduces the taser firing delay.

MrPerson committed:
Was this really necessary? Come on Metacide get your shit together.

Riesen committed:
fuck egun disablers

I hate paprika.

Riesen committed:
subtle message fix

Now we can finally RP as the Honkmother without them thinking we're Space Jesus.

Riesen committed:
fuck learning python

I'm editing this by hand anyways nerd.

gcoremans committed:
informative commit name

proper, informative extended description

Riesen committed:
no baton for CE

he doesn't deserve such a fine piece of hardware

Riesen committed:
removes baton for CMO

he has a hypospray instead

Riesen committed:
removes baton from RD

he can use a bomb instead

Riesen committed:
Update .all_changelog.yml

Cheridan committed:
Merge pull request #8290 from Ahammer18/droneassfinal

Drone asses!

Riesen committed:
Update engineering.dm

Riesen committed:
Update .all_changelog.yml

Riesen committed:
Update .all_changelog.yml

Riesen committed:
Update .all_changelog.yml

MrPerson committed:
Moved all pull icon updating to update_icon(), to reduce copy+paste.
Made the pull icon update_icon() as part of creation as requested by oranges.

hornygranny committed:
Merge pull request #8279 from MrPerson/pull_rapidemente_ariba_ariba_vamanos

Pulling icon en-speedening

Cheridan committed:
Merge pull request #8300 from xxalpha/everyday

Fixes not being able to move from a single lattice.

Cheridan committed:
Merge pull request #8302 from phil235/DoorCrushMech

Door can now crush mech, dealing damage, just like with mobs.

Cheridan committed:
Merge pull request #8288 from Ahammer18/Sensors

Toggle/Adjust suit sensors

Cheridan committed:
Merge pull request #8247 from Incoming5643/Welcome_to_ireland

Explicitly passes a mobs mutant_color in blood samples

Cheridan committed:
Merge pull request #8256 from phil235/StrangenessTextFix

Strangeness text, janicart, alien nest, weak golem fixes

Cheridan committed:
Merge pull request #8236 from phil235/DoughChange

Dough change

Cheridan committed:
Merge pull request #8227 from phil235/HulkCritFix

Hulk crit fixes

Cheridan committed:
Merge pull request #8222 from Incoming5643/I_dont_even_know_who_I_am_anymore

Magic Mania 1.855: Appearances are everything

Skrylar committed:
Merge pull request #3 from Riesen/patch-2

subtle message nerf

Skrylar committed:
Merge pull request #6 from Riesen/patch-5

CMO baton nerf

Skrylar committed:
Merge pull request #5 from Riesen/patch-4

replace CE baton with RCD

Skrylar committed:
Merge pull request #2 from Riesen/patch-1

eguns shoot electrodes instead of disablers

Skrylar committed:
Merge pull request #7 from Riesen/patch-6

removes baton from RD

Skrylar committed:
Merge pull request #4 from Riesen/patch-3

manual changelog update

Cheridan committed:
Merge pull request #8257 from phil235/ReflectFix

Simplifies laser reflection code

Cheridan committed:
Merge pull request #8206 from RemieRichards/AILessProcessing

Hostile Mob AI Optimisation

phil235 committed:
Fix dough and cakebatter recipes.

phil235 committed:
Fix typo in catalyst.
Fix flamethrower crafting recipe (now only use 1 rod, calls checkparts() to set up the weldingtool and igniter).
Fix lack of message when trying to turn on an unsecured welding tool.
Fix lack of icon update when clicking plasma tank with empty flamethrower.

RemieRichards committed:
Merge pull request #8273 from Jordie0608/gamepanelmorelikelamepanel

Search button on game panel spawn menu

Cheridan committed:
Merge pull request #8296 from phil235/TablecraftFix

Tablecraft Fix and recipe detection

Cheridan committed:
Merge pull request #8286 from phil235/SteelcapFix

Steel cap, sandstone and welding fixes plus flashing refactor

Cheridan committed:
Merge pull request #8311 from phil235/SprayFix

Fixes chemsprayer duplicating reagents. Simplifying its code.

Cheridan committed:
Merge pull request #8255 from GunHog/SiliconVerbFix

Fixes "Sensor Augmentation" showing in silicon verbs

Jordie0608 committed:
adds and edits ymls

Jordie0608 committed:
changelog compiled

Jordie0608 committed:
Merge pull request #8353 from Jordie0608/ninjasarelame

Changelog

Cheridan committed:
Merge pull request #8334 from MrPerson/cleanup_crew_comin_through

Code cleanup

Cheridan committed:
Merge pull request #8310 from phil235/RussianRevolverFix

Fixes russian revolver's special mechanism message

Cheridan committed:
Merge pull request #8328 from Miauw62/asdfjo

Chat toggles are saved to the savefile properly again.

Cheridan committed:
Merge pull request #8319 from MMMiracles/bearpelts4everyone

Ability to put bears on meat spikes.

phil235 committed:
Merge branch 'master' of https://github.com/tgstation/-tg-station into BurgerSpriteFix

Conflicts:
	icons/obj/food.dmi

Razharas committed:
Merge pull request #8348 from phil235/DoughHotfix

Fix dough and cakebatter recipes.

Cheridan committed:
Merge pull request #8204 from Iamgoofball/patch-41

Nerfs acid blob again

phil235 committed:
Merge pull request #8113 from Xhuis/boom_boom_shake_the_room

Restores gibbing for suicide C4

phil235 committed:
Merge pull request #8173 from AnturK/flagfix

Fixes pref flags

Razharas committed:
Merge pull request #8309 from phil235/BurgerSpriteFix

Fixes burger and sandwich ingredient stacking and sprites.

Skrylar committed:
Time to update /tg/ yet again.

Conflicts:
	html/changelogs/.all_changelog.yml
